### PR TITLE
Management of two new default pages: terms and privacy

### PR DESCRIPTION
**Pull request by donker**
Issue description here:

https://github.com/dnnsoftware/Dnn.Platform/issues/2357

### CODE REVIEW

File: Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx [hunk 1]
```diff
@@ -90,7 +90,8 @@ class DefaultPagesSettingsPanelBody extends Component {
         let defaultPagesSettings = Object.assign({}, state.defaultPagesSettings);
 
         if (key === "SplashTabId" || key === "HomeTabId" || key === "LoginTabId" || key === "RegisterTabId" ||
-            key === "UserTabId" || key === "SearchTabId" || key === "Custom404TabId" || key === "Custom500TabId") {
+            key === "UserTabId" || key === "SearchTabId" || key === "Custom404TabId" || key === "Custom500TabId" ||
```
Comment (ohine) [inline Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx]: I'd prefer this to be swapped out to a switch statement due to the number of conditions, however that's my personal preference on coding styles and I don't think it's a requirement to merge if the second reviewer agrees.

File: Extensions/Settings/Dnn.PersonaBar.SiteSettings/admin/personaBar/App_LocalResources/SiteSettings.resx [hunk 2]
```diff
@@ -1072,7 +1072,7 @@
     <value>Enable Localized Content</value>
   </data>
   <data name="EnableLocalizedContentHelpText.Text" xml:space="preserve">
-    <value>Enabling localized content allows you to provide translated module content in addition to displaying translated static text. Once localized contetnt is enabled the default site culture will be permanently set and <strong>cannot be changed</strong>.</value>
+    <value />
```
Comment (ohine) [inline Extensions/Settings/Dnn.PersonaBar.SiteSettings/admin/personaBar/App_LocalResources/SiteSettings.resx]: Was this an intentional change as well?

Comment (donker) [reply Extensions/Settings/Dnn.PersonaBar.SiteSettings/admin/personaBar/App_LocalResources/SiteSettings.resx]: Good catch. Updating ...

File: Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx (hunk 1 above)
Comment (donker) [reply Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx]: I went with what was there before. But totally agree.

Comment (donker) [reply Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/defaultPagesSettings/index.jsx]: Just uploaded a switch statement instead

**Review by mitchelsellers — APPROVED** [review]
LGTM  Appreciate the changed layout to a switch statement!

**Comment by mitchelsellers** [comment]
@donker Is this outdated at this point?  Does the CookieConsent one get this all?

**Comment by mitchelsellers** [comment]
@donker Can we get you to adjust to resolve the merge issues?

**Comment by valadas** [comment]
@donker I think this got done in another PR right? I have seen and tested those two new special pages in 9.3.0.103

**Comment by donker** [comment]
Yes, although it's a bit of a mess now that this code actually forked from the PR that was approved elsewhere. We'll need to improve procedures.

**Pull request closed without merging**